doc/openocd.texi

```diff
@@ -11369,16 +11369,24 @@ OpenOCD. When off, they generate a breakpoint exception handled internally.
 The commands below can be used to prevent OpenOCD from using certain RISC-V trigger features.
 For example in cases when there are known issues in the target hardware.
 
-@deffn {Command} {riscv set_enable_eq_match_trigger} [on|off]
-When on (default), allow OpenOCD to use exact-match triggers in watchpoints.
-@end deffn
+@deffn {Command} {riscv set_enable_trigger_type} [(@option{eq}|@option{napot}|@option{ge_lt}|@option{all}) (@option{wp}|@option{none})]
+Control usage of triggers in wp by OpenOCD. All trigger types are enabled by default.
+This command affects only on wp which will be set after the command.
```

Comment:
I have tried to improve the wording so that it is even clearer to the reader.

Please, check if I understood the original text properly and rephrased it accordingly. Thanks.

```suggestion
Control which RISC-V trigger features can be used by OpenOCD placing watchpoints.
All trigger features are allowed by default. Only new watchpoints, inserted after this command, 
are affected (watchpoints that were already placed before are not changed).
```